back/src/Guide/Database/Queries/Insert.hs

```diff
@@ -100,6 +103,51 @@ insertCategoryWithCategoryRow categoryRow = do
         |]
   lift $ HT.statement categoryRow statement
 
+insertCategory2
```

Comment:
You can comment out the old code instead of leaving it around and adding `2` to the new functions. Since the old schemas are commented out, I think it's alright to also comment out the old updates/selects/etc.